### PR TITLE
Add document for IPv6 support.

### PR DESCRIPTION
**Pull request by cathay4t**
* Use 2001:DB8::/32 block for document purpose.
 * Mention the current limitation of IPv6 support:
    * Not support flipping status.
    * Ignore IPv6 link local address.

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+<!-- vim-markdown-toc GFM -->
```
Comment (EdDev) [inline examples.md]: What does this do? ""

I like having a ToC. It just fits a separate PR.

File: examples.md [hunk 2]
```diff
@@ -39,7 +53,7 @@ interfaces:
 The example includes 3 ethernet interfaces, two with static IPv4 and the 3rd
 with no IP and with the state set to down.
 
-```
+```yaml
```
Comment (EdDev) [inline examples.md]: All examples are yaml, we do not need to mention it.

File: index.md [hunk 3]
```diff
@@ -111,4 +111,9 @@ Want to see nmstate in action? Check out the [demos](./demos.md).
 
 ## Stack support:
 - [IPv4 addresses (static)](./examples.md#interfaces-ethernet)
+- [IPv6 addresses (static)](./examples.md#interfaces-ethernet)
```
Comment (EdDev) [inline index.md]: Pointing to the same location is weird, how about just editing line 113 with `IPv4/6 addresses ...`?

No need to specify the limitation of IPv6 here, you can add a note in the example itself. Same for the link-local info.
(We actually need a man or user guide to explain each property, the command line, the behaviour and so on. Unfortunately we do not have capacity for that at the moment.)

**Review by EdDev — CHANGES_REQUESTED** [review]

File: index.md [hunk 4]
```diff
@@ -111,4 +111,9 @@ Want to see nmstate in action? Check out the [demos](./demos.md).
...
+    - Fliping IPv6 status(enable/disable) is not supported yet.
```
Comment (EdDev) [inline index.md]: We do not support setting IPv6 stack to disable. You can point to the Jira issue that tracks this.

File: examples.md (hunk 1 above)
Comment (cathay4t) [reply examples.md]: It's just a VIM plugin which automatically update TOC.
OK. I will remove it. I don't want to spend that time discussing that thing.

File: examples.md (hunk 2 above)
Comment (cathay4t) [reply examples.md]: It's for syntax highlight. I will remove it.

**Comment by cathay4t** [comment]
New patch uploaded as suggested.

File: examples.md (hunk 2 above)
Comment (tyll) [reply examples.md]: Good catch @cathay4t, IMHO it would be better to actually add it everywhere If it improves the syntax hightlighting.

Comment (EdDev) [reply examples.md]: Nice, I did not know it is for highlight.
It will be nice to see it everywhere in a new PR.

**Review by EdDev — APPROVED** [review]

File: examples.md [hunk 5]
```diff
@@ -36,8 +36,8 @@ interfaces:
 ```
 
 ## Interfaces: ethernet
-The example includes 3 ethernet interfaces, two with static IPv4 and the 3rd
-with no IP and with the state set to down.
+The example includes 3 ethernet interfaces, two with static IPv4 and IPv6 and
```
Comment (EdDev) [inline examples.md:39]: I think the last `and` should be replaced with a `,`.

**Pull request merged**